Database/Corrections/sodQuestFixes.lua

```diff
@@ -2358,6 +2358,33 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -424992,
             [questKeys.zoneOrSort] = sortKeys.ROGUE,
         },
+        [90139] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {{1161,1162,1163,1164}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.WARRIOR,
+            [questKeys.objectivesText] = {"Kill Troggs for the Skull-Shaped Geode. Then get hit by a Skullthumper's stun and use the geode to loot the rune."},
+            [questKeys.requiredSpell] = -425443,
+            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
+        },
+        [90140] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {
+                ['objectStart'] = {408799},
+                ['creatureStart'] = {504,95,449,590,589,121,285,46,732,735,210487},
+            },
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
```

Comment:
[Gillsbane](https://www.wowhead.com/classic/item=209420/gillsbane) requires you to be level 14 to be equipped. But in order to complete this quest you need to defeat the [Horror of the Deep](https://www.wowhead.com/classic/npc=210487/horror-of-the-deep) which is a level 20 enemy. So I'd say we increase this quest to level 20.

I wouldn't want to touch the `requiredLevel` as you can still "start" this rune by looting the hammer even if you are not level 14.

```suggestion
            [questKeys.questLevel] = 20,
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5435/commits/c8e3bc6a9464efb7fa7855ffb6cc673706cf3673

I was just about to check that as `90141`, `90142`, `90143` all suffer from the same issue where the start mobs for it are 12-14, but the last enemy is 20.

Reply:
Good finding, yeah Warlocks need the same mob. Feel free to adjust these fake quests in a different PR 👍🏻